test/unit/stagedProposalProcessor/concrete/reportProposalResult/reportProposalResult.tree

```diff
@@ -3,18 +3,49 @@ ReportProposalResult_SPP_UnitTest
 │   ├── when voteDuration has not passed
 │   │   ├── when the caller is an allowed body
 │   │   │   ├── when should try advance stage
-│   │   │   │   ├── it should record the result.
-│   │   │   │   ├── it should emit event.
-│   │   │   │   └── it should call advanceProposal function.
+│   │   │   │   ├── when proposal can be advanced
+│   │   │   │   │   ├── it should record the result.
+│   │   │   │   │   ├── it should emit event proposal result reported.
+│   │   │   │   │   └── it should call advanceProposal function and emit event.
+│   │   │   │   └── when proposal can not be advanced
+│   │   │   │       ├── it should record the result.
+│   │   │   │       ├── it should emit event proposal result reported.
+│   │   │   │       └── it should not call advanceProposal function nor emit event.
 │   │   │   └── when should not try advance stage
 │   │   │       ├── it should record the result.
 │   │   │       ├── it should emit event.
 │   │   │       └── it should not call advanceProposal function.
 │   │   └── when the caller is not an allowed body
 │   │       ├── it should not record the result in the right proposal path.
-│   │       └── it should record the result for historical data. 
+│   │       └── it should record the result for historical data.
 │   └── when voteDuration has passed
 │       ├── it should record the result.
 │       └── it should emit event.
 └── given non existent proposal
-    └── it should revert.  
\ No newline at end of file
+    └── it should revert.
+
+
+// # given existent proposal
```

Comment:
What is this ?

Reply:
I use that to generate the tree is easier for me, and I forgot to remove it -_-

removed in https://github.com/aragon/staged-proposal-processor-plugin/pull/13/commits/eee3722446cb807602a0cd3981550dad04153196